lib/styleDictionary/util.js

```diff
@@ -0,0 +1,98 @@
+const DEFAULT_KEY = 'default';
```

Comment:
Why does this need to be scoped at top when it is only used once?

Reply:
I try to avoid using string and number literals whenever I can (and think about it).

Reply:
I know, but it can be scoped to createTree... function since it is only used there.